test/runtests.jl

```diff
@@ -52,6 +52,8 @@ include("common.jl")
 
     include("multi-nw-cnd.jl")
 
+    include("bt.jl")
```

Comment:
Call this file `util` or `obbt`, so it matches the other file structures.

Reply:
changed to util.jl